database/migrations/model/Version20220328170502.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Migrations\Model;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+
+class Version20220328170502 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $sql = <<<SQL
+UPDATE `SummitScheduleFilterElementConfig` SET Label = 'Category Groups' WHERE Type = 'TRACK_GROUPS';
```

Comment:
this is not rite u should only update it if the Former label value was 'Categories Groups' otherwise u will overwrite legit customizations
regards

Reply:
Done